ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -167,14 +167,18 @@ mlArg
 
 /** clauses */
 fromClause
-    : SOURCE EQUAL tableSourceClause
-    | INDEX EQUAL tableSourceClause
+    : SOURCE EQUAL clusterTableSourceClause
```

Comment:
why not reuse tableSourceClause?

Reply:
so that the tableSourceClause (without cluster) can be reserved for Describe command.
I don't want to let user query `describe cluster-1:index-name` when we can't really fetch the field mapping at the remote cluster.

Reply:
how about `describe *:index-name`? does it works?

Reply:
It does not work.
My rationale is that since we don't have cross cluster field mapping request, we should disallow the : syntax for `describe` completely to avoid confusion.

Reply:
describe *:index-name should work, right? with same limitation as search

Reply:
change it so that ":" in describe is allowed